.cirrus.yml

```diff
@@ -7,11 +7,11 @@ flutter_docker_builder:
     # from https://flutter.dev/docs/development/tools/sdk/releases
     matrix:
     - DOCKER_TAG: latest
-      FLUTTER_VERSION: v1.9.1+hotfix.2
+      FLUTTER_VERSION: v1.9.1+hotfix.4
     - DOCKER_TAG: stable
-      FLUTTER_VERSION: v1.9.1+hotfix.2
+      FLUTTER_VERSION: v1.9.1+hotfix.4
     - DOCKER_TAG: beta
-      FLUTTER_VERSION: v1.9.1+hotfix.2
+      FLUTTER_VERSION: v1.9.1+hotfix.4
     - DOCKER_TAG: dev
       FLUTTER_VERSION: v1.10.3
```

Comment:
Dev should be bumped to 1.10.6. as well.

Reply:
@fkorotkov I did https://github.com/cirruslabs/docker-images-flutter/pull/32/commits/8435212103e1a2cf06d18e839c377200788c5a67